compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -318,6 +270,8 @@ def IREELinalgExt_GatherOp : IREELinalgExt_Op<"gather",
   let hasCanonicalizer = 1;
 }
 
+// map_scatter op does not have enough operands to use AttrSizedOperandSegments trait
+// so we inherit from IREELinalgExt_PureOp for now.
```

Comment:
The comment is not accurate to me. The reason of using AttrSizedOperandSegments for an op if it has at least two variadic operands. I think you can drop the comments for this case and other cases. I'll do a cleanup for this in a follow-up. Thanks.

https://mlir.llvm.org/docs/DefiningDialects/Operations/#variadic-operands